docs/design/lexical_conventions/operator_tokens.md

```diff
@@ -0,0 +1,61 @@
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# Operator Tokens
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Overview](#overview)
+-   [Tokens](#tokens)
+-   [Alternatives Considered](#alternatives-considered)
+-   [References](#references)
+
+<!-- tocstop -->
+
+## Overview
+
+An _operator_ is a mathematical/logical symbol that is useful for operator
+overloading like `+`, `*`, `<`, and more.
```

Comment:
With the change to symbolic tokens, this needs to be updated. Operators are one use of symbolic tokens, but they are also used in patterns (`:`), declarations (`->` to indicate return type, `,` to separate parameters), statements (`;`, `=`, and so on), and other places (`,` to separate function call arguments).